### PR TITLE
Prepare Release 1.3.3

### PR DESCRIPTION
**Pull request by jhalliday**

### CODE REVIEW

**Comment by jhalliday** [comment]
@tigrannajaryan draft changelog, as per discussion on slack.  I'm assuming we're going with 1.3.3 rather than 1.4, since the substantive changes are only in the experimental profiling proto, not the stable protocols.

**Review by jmacd — APPROVED** [review]

**Review by reyang — APPROVED** [review]

**Review by tigrannajaryan — APPROVED** [review]

**Comment by tigrannajaryan** [comment]
Draft release ready: https://github.com/open-telemetry/opentelemetry-proto/releases/tag/untagged-4e0fe3e6635fb54a1f02

I will keep this PR open until tomorrow and will merge, then release if I don't see objections.

Please don't merge other PRs until then so that we don't have to update the CHANGELOG again.

**Comment by tigrannajaryan** [comment]
Follow up to correct the version number in CHANGELOG: https://github.com/open-telemetry/opentelemetry-proto/pull/601

**Pull request merged**
